pkg/api/schema.graphql

```diff
@@ -0,0 +1,83 @@
+schema {
+    query: Query
+}
+
+# Leave date as a string (specify YYYY-MM-DD) and convert later?
+# How do we do latest/earliest?
+#scalar Date
+
+# Query contains the entry points into the schema
+type Query {
+    #bug(id: Int!, datestamp: String = "latest"): Bug
+    # TODO: Add sorting argument for bugs? asc/desc + id/assigned_to/component/etc/etc...default to ???
+    bugs(datestamp: String = "_latest", components: [String!]): [Bug]!
+    snapshot(datestamp: String = "_latest"): Snapshot
+    # Should this just be "graphdata"? All lowercase vs camelcase?
+    graphData(components: [String!]): GraphData
+    release(name: String!, components: [String!]): Release
+
+    # Return lists of things?  As of now, no...
+    #snapshots(): [String]! # list of datestamps
+    #releases(): [String]! # list of release names
+    #components(): [String]! # list of valid components...maybe?
+}
+
+type Bug {
+    # Should this use scalar type ID?  (treated as string, but not intended to be human readable...)
```

Comment:
Don't see running into any problems with id as an int, string, or graphql.ID.  Its simple to pick one and cast appropriately...does it matter?